recipe/meta.yaml

```diff
@@ -1,42 +1,41 @@
 {% set name = "h3" %}
-{% set version = "3.7.4" %}
-{% set sha256 = "f8edf5a546b31afdcd801b60448ea890ce1ff418fb784335e1329519f13aa85e" %}
+{% set version = "3.7.6" %}
 
 package:
   name: {{ name|lower }}-py
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 9bbd3dbac99532fa521d7d2e288ff55877bea3223b070f659ed7b5f8f1f213eb
 
 build:
   number: 0
-  skip: true # [s390x]
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - make                  # [unix]
+    - cmake                 # [unix]
```

Comment:
what about windows, is cmake implicit there ?

Reply:
I enabled `cmake` on `win64` in `build` but then it will be in both environments (`build` and `host`). Looking at the logs, I found that `cmake` is invoked by `scikit-build` from `host` on all platforms.
Finally, I moved it to `host` and left a comment.